_posts/2019-07-04-tip-autoconnect-order.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Automatic Connections to New Nodes"
+subtitle: "Tip of the Day"
+category: tips
+tags: [nodes]
+---
+
+<img width="200" class="float-left" style="margin-right: 20px;" src="{{ site.baseurl }}/resources/images/2019-07-04-tip-autoconnect-order/spaghetti.png" alt="Spaghetti">
```

Comment:
Blank line after HTML, to make the Markdown more readable.

Reply:
Yeah, but then it splits it into 'press to read more' which this really doesn't need...